aries-site/src/examples/components/checkbox/CheckBoxDescription.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { Box, CheckBox, FormField } from 'grommet';
+
+export const CheckBoxDescriptionExample = () => {
+  const [checked, setChecked] = useState();
+
+  return (
+    <Box>
+      <FormField
+        htmlFor="checkbox-desc"
+        label="Checkbox Heading"
```

Comment:
I think we should add 'name' to both FormField and CheckBox to promote best practice. The name will be needed when used within a form.

See https://v2.grommet.io/form as example (however, these examples need the htmlFor and id props added like you have here)